diff -aur a/ezanvakti/etc/ayarlar.in b/ezanvakti/etc/ayarlar.in
--- a/ezanvakti/etc/ayarlar.in	2017-02-01 08:48:14.000000000 +0300
+++ b/ezanvakti/etc/ayarlar.in	2017-02-03 16:14:55.107008990 +0300
@@ -189,16 +189,16 @@
 
 
 # Printf arayuz yazı biçimi
-GELISMIS_ARAYUZ_BICIMI='%-62s%s\n%s\n%s\n\t\t\t\t%s\n%s\n\n%s\n\n\t\t\t %s\n\t\t\t %s'
+GELISMIS_ARAYUZ_BICIMI='%-69s%s\n%s\n%s\n\t\t\t\t\t%s\n%s\n\n%s\n\n\t\t\t\t%s\n\t\t\t\t%s'
 
 # Printf conky çıktısı biçimi
 CONKY_BICIMI='%-14s%s\n%-15s%s\n%-17s%s\n%-17s%s\n%-14s%s\n%-17s%s\n'
 
 # Printf arayüzlerdeki vakit çıktısı biçimi
-VAKIT_BICIMI='\t\t\t%-14s%s\n\t\t\t%-16s%s\n\t\t\t%-18s%s\n\t\t\t%-18s%s\n\t\t\t%-14s%s\n\t\t\t%-18s%s'
+VAKIT_BICIMI='\t\t\t\t%-13s%s\n\t\t\t\t%-15s%s\n\t\t\t\t%-17s%s\n\t\t\t\t%-16s%s\n\t\t\t\t%-12s%s\n\t\t\t\t%-17s%s'
 
 # Arayüz çizgi simgesi
 CIZGI_SIMGESI='@'
 
 # Arayüz çizgi karakter uzunluğu
-CIZGI_UZUNLUGU=26
+CIZGI_UZUNLUGU=22
